### PR TITLE
Issue #450 fix

### PR DESCRIPTION
**Pull request by Rkamath2**
### Proposed Changes
#450 
  - Added filter to API for exercise comments (exercises/views.py)
  -
  -

### Please check that the PR fulfills these requirements

- [Yes ] Tests for the changes have been added (for bug fixes / features)
- [ Yes] New python code has been linted with with flake8 (``flake8 --config .github/linters/.flake8``)
and isort (``isort``) 
- [ Yes] Added yourself to AUTHORS.rst

### Other questions

* Does this PR introduce a breaking change such as a database migration? (i.e.
what changes might users need to make in their running application due to
this PR?)
No

### CODE REVIEW

File: wger/exercises/api/views.py [hunk 1]
```diff
@@ -221,11 +221,22 @@ class ExerciseCommentViewSet(viewsets.ReadOnlyModelViewSet):
...
-                        'exercise')
+    filterset_fields = ('exercise',
+                        'comment'
+                        )
+    """
+    Filter by language for exercise comments
+    """
+    def get_queryset(self):
+        qs = ExerciseComment.objects.all()
+        language = self.request.query_params.get('language', None)
```
Comment (rolandgeider) [inline wger/exercises/api/views.py:235]: You don't need to explicitly add None, .get() returns None by default

File: wger/exercises/api/views.py [hunk 2]
```diff
@@ -221,11 +221,22 @@ class ExerciseCommentViewSet(viewsets.ReadOnlyModelViewSet):
...
     """
-    queryset = ExerciseComment.objects.all()
     serializer_class = ExerciseCommentSerializer
+
     ordering_fields = '__all__'
-    filterset_fields = ('comment',
-                        'exercise')
+    filterset_fields = ('exercise',
+                        'comment'
+                        )
```
Comment (rolandgeider) [inline wger/exercises/api/views.py:229]: Why change the order here? :)

Comment (Rkamath2) [reply wger/exercises/api/views.py]: Sorry! Didnt realise I did that. Fixed it now

File: wger/exercises/api/views.py (hunk 1 above)
Comment (Rkamath2) [reply wger/exercises/api/views.py]: Corrected this.

**Comment by rolandgeider** [comment]
Looks good! Thanks!

**Pull request merged**